doc/api.md

```diff
@@ -127,7 +127,8 @@ RulerSinks defines a set of sinks for Events Ruler
 
 | Field | Description | Scheme | Required |
 | ----- | ----------- | ------ | -------- |
-| alertmanager | Alertmanager is an alertmanager sink to which only alerts can sink. | *[RulerAlertmanagerSink](#ruleralertmanagersink) | false |
+| alertmanagers | Alertmanagers are alertmanager sinks to which alerts can sink. | []*[RulerAlertmanagerSink](#ruleralertmanagersink) | false |
```

Comment:
`Alertmanagers are alertmanager sinks to which alerts can sink`
=>
`Alertmanager sinks to which events alerts will be sent`